### DISCUSSION TITLE
not updating after yarn start

### DISCUSSION THREAD

**Discussion by soonki-kang** [iamWing/electron-react-typescript-base-proj#43] [Q&A]
react app not updating after yarn start
how to change scripts in package.json
thanks for your help

**Comment by iamWing** [comment]
> react app not updating after yarn start
> how to change scripts in package.json
> thanks for your help

@soonki-kang Can you please tell me what's not updating? Have you run `yarn dev`(not sure if it's the correct syntax, I mainly use `npm`) before running `yarn start`? I'm not sure how to help you with no details provided.

As for the scripts, just change them in `package.json` if needed. Guess you can just replace `npm run` with `yarn` if that's what you wanna do.
